QuickTour/QuickTour.tex

```diff
@@ -759,10 +759,10 @@ \section{Defining a new method}
 \index{SUnit}
 
 \needlines{3}
```

Comment:
Something is odd here. Two needlines after another? Also why did the number of lines increase to 9?

Reply:
See f8a32a5.